### ISSUE TITLE
Update to 1.20

### ISSUE THREAD

**Issue by gitdrug** [Andrew6rant/InventoryTabs#98]
Update to 1.20

**Comment by Goldsucher07** [comment]
yes

**Comment by Belu97** [comment]
YEAASSSSS

**Comment by Guerdi** [comment]
YEAAAAAS

**Comment by MendingNights** [comment]
YES!  This is the last mod I need updated.  It's always the last one, every update.

**Comment by MTMomo99** [comment]
PLEASE UPDATE IT TO 1.20.1

**Comment by CodeF53** [comment]
Please stop sending messages. He will update when he updates.

I don't need a new email from you guys begging for it to happen every couple of days

**Comment by MendingNights** [comment]
> Please stop sending messages. He will update when he updates.
> 
> I don't need a new email from you guys begging for it to happen every couple of days

Sometimes people need a nudge to get things done.  Turn off notifications if it means that much to you.  Until then, deal with it.

**Comment by Guerdi** [comment]
For those who are interested, here is an 1.20 version, updated by spiralhalo (https://github.com/spiralhalo/InventoryTabs/tree/1.20-port/)
[inventorytabs-0.9.0-beta-1.20.zip](https://github.com/Andrew6rant/InventoryTabs/files/11901048/inventorytabs-0.9.0-beta-1.20.zip)

**Comment by skibidibada** [comment]
> For those who are interested, here is an 1.20 version, updated by spiralhalo (https://github.com/spiralhalo/InventoryTabs/tree/1.20-port/) [inventorytabs-0.9.0-beta-1.20.zip](https://github.com/Andrew6rant/InventoryTabs/files/11901048/inventorytabs-0.9.0-beta-1.20.zip)

this is only for 1.20 snapshot, not 1.20 or 1.20.1

**Comment by MendingNights** [comment]
> > For those who are interested, here is an 1.20 version, updated by spiralhalo (https://github.com/spiralhalo/InventoryTabs/tree/1.20-port/) [inventorytabs-0.9.0-beta-1.20.zip](https://github.com/Andrew6rant/InventoryTabs/files/11901048/inventorytabs-0.9.0-beta-1.20.zip)
> 
> this is only for 1.20 snapshot, not 1.20 or 1.20.1

Incorrect.  It works on 1.20.1.  I'm currently using it on 1.20.1 and it works perfect.
